lint381/c.py

```diff
@@ -11,6 +11,20 @@
 
 
 @linter.register
+@with_matched_tokens(start=match_regex("^sizeof$"), end=match_regex("^char$"))
+def prohibited_unsigned_char(source, *, match):
```

Comment:
This doesn't actually have anything to do with prohibited `unsigned char`s. Perhaps just `sizeof_char` or something similar?